docusaurus/docs/introduction/data-frames.md

```diff
@@ -87,6 +87,10 @@ Let's look at an example. The following table demonstrates a data frame with two
 
 Each field has three values, and each value in a field must share the same type. In this case, all values in the `time` field are timestamps, and all values in the `temperature` field are numbers.
 
+While time fields represent timestamps, the type of the values should be `Number` (TypeScript) or `time.Time` (Golang).
+
+In the plugin frontend code, it's possible to convert other formats to `Number` using the function []`ensureTimeField`](https://github.com/grafana/grafana/blob/v10.1.5/packages/grafana-data/src/transformations/transformers/convertFieldType.ts#L231-L243) from the `@grafana/data` package. This function converts strings following the ISO 8601 format (for example, `2017-07-19 00:00:00.000`), Javascript `DateTime`s and strings with relative times (for example, `now-10s`) to `Numbers`.
+
 One restriction on data frames is that all fields in the frame must be of the same length to be a valid data frame.
```

Comment:
Since we were talking about the time restriction, I would start this paragraph with "Another restriction on data frames ..."

Reply:
edited accordingly.